### PR TITLE
Results for 2022 Steering Elections

### PR DESCRIPTION
**Pull request by dims**
That's a wrap folks! Congrats to the new steering members.

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: elections/steering/2022/results.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Results of the 2022 Steering Committee Election
```
Comment (mrbobbytables) [inline elections/steering/2022/results.md:1]: It looks like we skipped it for last year's election (think it was an accident) - but in the past we've published the full list in the event we do need to fill an additional seat:
- https://github.com/kubernetes/community/blob/master/elections/steering/2020/RESULTS.md#results
- https://github.com/kubernetes/community/blob/master/elections/steering/2019/RESULTS.md#results
- https://github.com/kubernetes/community/blob/master/elections/steering/2018/RESULTS.md#results

Comment (coderanger) [reply elections/steering/2022/results.md:1]: Maybe do the next one or two? I don't want the results to go over into hurt feelings territory if we can help it.

Comment (dims) [reply elections/steering/2022/results.md:1]: @coderanger dunno, i feel like picking full list vs only winners (not winners + a couple more)

Comment (justaugustus) [reply elections/steering/2022/results.md:1]: Full list for transparency, please.

**Review by coderanger — APPROVED** [review]
/lgtm
/approve

**Pull request merged**